### ISSUE TITLE
Slack links (https://slack-redir.net/) not skipped

### ISSUE THREAD

**Issue by lopis** [sblask/webextension-skip-redirect#121]
### New Issue Checklist (tick off like this: `[x]`)

- [x] I checked existing issues (open and closed) for possible duplicates
- [x] I can reproduce the problem on the latest stable version (not nightly!)
- [ ] If you have a problem a specific website: adding the domain to your blacklist does not solve the problem
- [x] The issue is not about adding entries to the default blacklist

### What is your browser?

- [x] Firefox
- [ ] Chrome

### What is your operating system?

- [x] Linux
- [ ] Mac
- [ ] Windows

### Description (please include examples/screenshots where applicable)

Links originating from slack website don't seem to be affected by this addon. I always land on the https://slack-redir.net/ link before being redirected to the correct website.

**Comment by sblask** [comment]
I am sure that it works in general because I just tried and the redirect was skipped. It might be that you use blacklist mode and the link you clicked contains something from the blacklist. I use whitelist mode and I have `https://slack-redir.net/link*` on it.

**Comment by kowalcj0** [comment]
Hi,

@sblask thank you for this useful extension.

FYI. it also works fine in Vivaldi.

I like that lil trick with `307 Temporary redirect`. Simple yet effective :+1: 
![slack-redirect](https://user-images.githubusercontent.com/910451/69883775-8825dd80-12cd-11ea-9ff4-686794279bf4.gif)

Here's my config:
I've added `*://slack-redir.net/link?url=*` to `whitelist`:
![image](https://user-images.githubusercontent.com/910451/69883331-970b9080-12cb-11ea-94d9-9c732b05e69b.png)

PS. just to be 100% sure that slack redirects are blocked I've added `0.0.0.0 slack-redir.net` to `/etc/hosts`.
